types/ember-test-helpers/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
```

Comment:
DOH. I went looking for this b/c I thought it might be the issue; but apparently I just checked the wrong `tsconfig.json` file.

Reply:
Yep, that was it. The compiler has given me one hell of a time testing this locally. There's something seriously wrong with the incremental compiler/caching logic. I shouldn't have to dirty files to make stuff work